frontend/pages/hosts/details/cards/HostSummary/HostSummary.tsx

```diff
@@ -218,50 +220,65 @@ const HostSummary = ({
   };
 
   const renderIssues = () => (
-    <div className="info-flex__item info-flex__item--title">
-      <span className="info-flex__header">
-        Issues{isSandboxMode && <PremiumFeatureIconWithTooltip />}
-      </span>
-      <span className="info-flex__data">
-        <span
-          className="host-issue tooltip tooltip__tooltip-icon"
-          data-tip
-          data-for="host-issue-count"
-          data-tip-disable={false}
-        >
-          <Icon name="error-outline" color="ui-fleet-black-50" />
-        </span>
-        <ReactTooltip
-          place="bottom"
-          effect="solid"
-          backgroundColor={COLORS["tooltip-bg"]}
-          id="host-issue-count"
-          data-html
-        >
-          <span className={`tooltip__tooltip-text`}>
-            Failing policies ({summaryData.issues.failing_policies_count})
+    <DataSet
+      title={<>Issues{isSandboxMode && <PremiumFeatureIconWithTooltip />}</>}
+      value={
+        <>
+          <span
+            className="host-issue tooltip tooltip__tooltip-icon"
+            data-tip
+            data-for="host-issue-count"
+            data-tip-disable={false}
+          >
+            <Icon name="error-outline" color="ui-fleet-black-50" />
           </span>
-        </ReactTooltip>
-        <span className="info-flex__data__text">
-          {summaryData.issues.total_issues_count}
-        </span>
-      </span>
-    </div>
+          <ReactTooltip
+            place="bottom"
+            effect="solid"
+            backgroundColor={COLORS["tooltip-bg"]}
+            id="host-issue-count"
+            data-html
+          >
+            <span className={`tooltip__tooltip-text`}>
+              Failing policies ({summaryData.issues.failing_policies_count})
+            </span>
+          </ReactTooltip>
+          <span>{summaryData.issues.total_issues_count}</span>
+        </>
+      }
+    />
   );
 
   const renderHostTeam = () => (
-    <div className="info-flex__item info-flex__item--title">
-      <span className="info-flex__header">Team</span>
-      <span className={`info-flex__data`}>
-        {summaryData.team_name ? (
+    <DataSet
+      title="Team"
+      value={
+        summaryData.team_name !== "---" ? (
```

Comment:
Can this potentially be undefined (not loaded) or null ([interface](https://github.com/fleetdm/fleet/blob/d50ee081f20366740acd21fa23a77edb6c1f1518/frontend/interfaces/host.ts#L288))?

We should probably define better typing than `any` for `summaryData` too

Reply:
Good question, I'll look into it

Reply:
Found a ticket here: https://github.com/fleetdm/fleet/issues/7457